libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -422,6 +423,22 @@ internal object MapboxNavigationTelemetry {
         userFeedbackCallbacks.remove(userFeedbackCallback)
     }
 
+    @ExperimentalPreviewMapboxNavigationAPI
```

Comment:
How do you choose when to add the annotation? The object is internal, doesn't it mean that it's not API?

Reply:
Good catch. I don't we need `@ExperimentalPreviewMapboxNavigationAPI` on this method.

Reply:
:+1